Action[] actions
string[] topics
State[] previousStates
State initialState
string stateMachine # json {edges: {state: [states], labels: "(state, state)": label, values: {state: point}}, empty if not model-refine mode
